src/main/java/io/confluent/connect/jdbc/util/JdbcUtils.java

```diff
@@ -202,6 +202,20 @@ public static boolean isColumnNullable(
     return false;
   }
 
```

Comment:
Looks like the other public methods in this class have a javadoc. Should we add one here too?